### DISCUSSION TITLE
After VM deletion and new vm spin up, in Flower UI, deleted vm still showing.

### DISCUSSION THREAD

**Discussion by rishi-kumar-gupta** [apache/airflow#31020] [General]
We had configure autoscalling in mig and installed airflow and flower, once instance deleted and new instance spin up, still deleted instance is showing in Flower UI.

**Comment by potiuk** [comment]
I guess it is a question to flower and you should open an issue there,  but I guess restarting it should help. We are merely spinning of flower here, not doing anything with it so any issues with flower should be directed to them.
